ext/js/data/options-util.js

```diff
@@ -1115,8 +1115,10 @@ export class OptionsUtil {
      */
     _updateVersion23(options) {
         for (const {options: profileOptions} of options.profiles) {
-            for (const dictionary of profileOptions.dictionaries) {
-                dictionary.partsOfSpeechFilter = true;
```

Comment:
Why is the if check needed?

Reply:
I see the comment in the OP but still don't quite understand, as it the length being greater than 0 should be irrelevant for the for loop?

Reply:
Without it:
```
 FAIL  test/options-util.test.js > FieldTemplatesUpdate > field-templates-update-test-13
TypeError: profileOptions.dictionaries is not iterable
 ❯ OptionsUtil._updateVersion24 ext/js/data/options-util.js:1134:57
    1132|
    1133|         for (const {options: profileOptions} of options.profiles) {
    1134|                 for (const dictionary of profileOptions.dictionaries) {
       |                                                         ^
    1135|                     dictionary.useDeinflections = true;
    1136|                 }
 ❯ OptionsUtil._applyUpdates ext/js/data/options-util.js:487:46
 ❯ OptionsUtil.update ext/js/data/options-util.js:100:19
 ❯ test/options-util.test.js:1678:52
```

Caused by this guy running through it (the other test dict works fine) <https://github.com/themoeway/yomitan/blob/b836fe7565de0ffd74663a37f88ff51efdc8b5ff/test/options-util.test.js#L107-L113>

It isn't actually checking for the length.  It's checking whether there is a length. The `> 0` bit isn't actually needed. I feel like theres probably a better way to do this but wasn't sure.

Reply:
You can use  `Array.isArray()`

Reply:
Changed it to use that now.